### PR TITLE
data:Junnygram

### PR DESCRIPTION
**Pull request by Junnygram**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5680"></a>

### CODE REVIEW

File: data/Junnygram.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+"name" : "Olaleye Oyewunmi",
+"type" : "personal",
+"bio" : "Physicist turned Frontend developer",
+"avatar" : "https://avatars.githubusercontent.com/u/90139191?v=4",
```
Comment (ChinmayMhatre) [inline data/Junnygram.json:5]:
```suggestion
```

Comment (ChinmayMhatre) [reply data/Junnygram.json:5]: You can remove this field as it isn't required anymore.

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Looks awesome! Left a comment for a change. Could you tell me what resource you followed to create the json file? someone's profile or documentation? Just asking as feedback.

**Comment by Junnygram** [comment]
> Looks awesome! Left a comment for a change. Could you tell me what resource you followed to create the json file? someone's profile or documentation? Just asking as feedback.

well its all in the starter guide, i just followed the template, did i do something wrong?

**Comment by ChinmayMhatre** [comment]
> > Looks awesome! Left a comment for a change. Could you tell me what resource you followed to create the json file? someone's profile or documentation? Just asking as feedback.
> 
> well its all in the starter guide, i just followed the template, did i do something wrong? 
> 

Not at all! I was wondering which guide had the avatar property. So that we could remove it from the documentation.

Please reopen this PR, the PR needs to be merged by the maintainer so that you can see your profile.

**Pull request closed without merging**